File: backend/src/main/java/com/liveticket/order/entity/Order.java
package com.liveticket.order.entity;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;
import lombok.Data;

import java.math.BigDecimal;
import java.time.LocalDateTime;

@Data
@TableName("lt_order")
public class Order {

    private Long id;
    private String orderNo;
    private Long userId;
    private Long eventId;
    private Long ticketSkuId;
    private Integer quantity;
    private BigDecimal unitPrice;
    private BigDecimal totalAmount;
    private Integer status;
    private Integer source;
    private String purchaseGuard;
    private LocalDateTime createdAt;
    private LocalDateTime paidAt;
    private LocalDateTime cancelledAt;
    private LocalDateTime updatedAt;
}

File: frontend/src/types/order.ts
export interface OrderVO {
  orderNo: string
  userId: number
  eventId: number
  eventTitle: string
  eventCoverUrl: string
  eventStartTime: string
  venueName: string
  ticketSkuId: number
  skuName: string
  quantity: number
  unitPrice: number
  totalAmount: number
  status: number
  source: number
  createdAt: string
  paidAt?: string | null
  cancelledAt?: string | null
}
